render-test/manifest_parser.cpp

```diff
@@ -412,6 +412,18 @@ mbgl::optional<Manifest> ManifestParser::parseManifest(const std::string& manife
         }
     }
 
+    auto filter = testFilter;
```

Comment:
instead could be `s/const std::string& testFilter/std::string testFilter/` in the method args